llama-index-integrations/vector_stores/llama-index-vector-stores-databricks/llama_index/vector_stores/databricks/base.py

```diff
@@ -152,6 +154,7 @@ def __init__(
         self._delta_sync_index_spec = index_description.delta_sync_index_spec
         self._direct_access_index_spec = index_description.direct_access_index_spec
         self._doc_id_to_pk = {}
+        self._node_info_columns = ["relationships", "start_char_idx", "end_char_idx"]
```

Comment:
nit: Let's define constants for these column names and "doc_id". They are used multiple times and typo prone.